lib/src/project.js

```diff
@@ -131,17 +135,21 @@ function project(location = {}, targetAltitude, previousResult) {
         result.options[k] = concat(result.options[k], v);
       }
     });
+    result.options.transitionForestType = transition.options.forestType;
```

Comment:
How about ...

```suggestion
    result.options.transitionForestType = sortForestTypes(transition.options.forestType);
```

... and removing the if statement below?

Reply:
Done, by including another check in the sortForestTypes function